services/QuillLMS/engines/comprehension/spec/controllers/comprehension/activities_controller_spec.rb

```diff
@@ -0,0 +1,192 @@
+require("rails_helper")
+
+module Comprehension
+  RSpec.describe(ActivitiesController, :type => :controller) do
+    before { @routes = Engine.routes }
+    context 'foo' do 
```

Comment:
runs correctly via `be rspec spec/controllers/comprehension/activities_controller_spec.rb:6 `